packages/web-components/@sfx/products/src/index.ts

```diff
@@ -0,0 +1,102 @@
+/**
+ * This test comment _supports_ [Markdown and other fun stuff](https://typedoc.org/guides/doccomments/)
+ */
+import { LitElement, customElement, css, html, property } from 'lit-element';
+
+@customElement('sfx-product')
+class Product extends LitElement {
+  @property({ type: String }) name = ''
+  @property({ type: String }) description = ''
+  @property({ type: Number }) price = ''
+  @property({ type: String }) imageSource = ''
+
+  static get properties() {
+    return {
+      name: { reflect: true },
+      description: { reflect: true },
+      price: { reflect: true },
+      imageSource: { reflect: true },
+    }
+  }
+
+  render() {
+    return html`
+      <style>
+        sfx-product {
+          padding: 5px;
+        }
+      </style>
+      <div>
+        <img src="${this.imageSource}" />
+        <p>${this.name}: $${this.price}.95</p>
+        <p>${this.description}</p>
+      </div>
+    `
+  }
+
+  createRenderRoot() {
+    return this;
+  }
+}
+
+@customElement('sfx-products')
+export class Products extends LitElement {
+  @property({ type: Number }) maxItems = 12;
+  @property({ type: Array }) products = [];
```

Comment:
This should be typed. We might end up in an unexpected type situation because TS has to infer what it's an array _of_ just from an empty array.